package.json

```diff
@@ -8538,6 +8538,258 @@
+                                                        "icon": "file",
+                                                        "type": "command",
+                                                        "command": "editor.action.formatDocument.multiple"
+                                                    },
+                                                    {
+                                                        "key": "S",
+                                                        "name": "+Format selection with formatter",
+                                                        "icon": "selection",
+                                                        "type": "command",
+                                                        "command": "editor.action.formatSelection.multiple"
+                                                    }
+                                                ]
+                                            },
+                                            {
+                                                "key": "a",
+                                                "name": "+Actions",
+                                                "icon": "zap",
+                                                "type": "bindings",
+                                                "bindings": [
+                                                    {
+                                                        "key": "b",
+                                                        "name": "Build workspace",
+                                                        "icon": "gear",
+                                                        "type": "command",
+                                                        "command": "zig-language-extras.buildWorkspace"
+                                                    },
+                                                    {
+                                                        "key": "d",
+                                                        "name": "Debug",
+                                                        "icon": "debug",
+                                                        "type": "command",
+                                                        "command": "zig.debug"
+                                                    },
+                                                    {
+                                                        "key": "r",
+                                                        "name": "Run workspace/file",
+                                                        "icon": "run",
+                                                        "type": "command",
+                                                        "command": "zig.run"
+                                                    }
+                                                ]
+                                            },
```

Comment:
This section doesn't follow the conventions described in https://vspacecode.github.io/docs/conventions#actions

Please change this section and all the other sections not following the convention

Reply:
Maybe move these to something like Run (`r`), Debug (`d`), Build (`c`)?